### DISCUSSION TITLE
Membuat shapefiles untuk Choroplet

### DISCUSSION THREAD

**Discussion by Saupia** [teamalgoritma/community#415] [Q&A] [answered]
Halo, saya mau membuat peta menggunakan Choroplet. Tetapi instruknis menyarankan untuk menggunakan shapefiles. Apaka ada saran cara menggunakan ini?

**Comment by ranydwi** [comment, marked as answer]
Dear Pak Renhard @Saupia ,

Untuk menggunakan Choropleth salah satu file yang dapat digunakan adalah shapefiles. Data shapefile adalah kumpulan data yang digunakan pada banyak aplikasi geospatial. Keuntungan dari data Shapefile adalah selain dapat diakses di banyak aplikasi, data Shapefile juga dapat dilihat dan diolah sedikit dalam mapshaper (apabila Bapak hanya ingin memakai geometry map suatu area tertentu atau melakukan ekstraksi ke dalam jenis file lainnya).

Referensi yang dapat saya berikan untuk bapak pelajari yaitu pada link berikut [choroplethR](https://rpubs.com/AlgoritmaAcademy/choroplethR)

Semoga membantu Pak.
Team Algoritma

**Reply by Saupia** [reply]
Ini masihh dalam tahap pencarian ya. 
Mohon paste link atau code jika ada jawaban.

Terima kasih

**Reply by ranydwi** [reply]
Dear Pak @Saupia, 
Dari link yang saya sudah berikan, silahkan Pak Reinhard mengunduh terlebih dahulu datanya dari GDAM untuk file shpnya [GADM untuk download peta](https://gadm.org/download_country_v3.html).

Kemudian silahkan ikuti instruksi yang ada pada [link rpubsnya](https://rpubs.com/AlgoritmaAcademy/choroplethR) Pak dimulai dari membaca data terlebih dahulu. Nanti untuk code detailnya silahkan dapat diatur sesuai dengan data dan variabel yang Bapak miliki.

Kalau ada masalah terkait code atau apapun yang berhubungan dengan choropleth ini silahkan ditanyakan kembali ya Pak.

Semoga membantu
Team Algoritma.

**Reply by Saupia** [reply]
Rani, 
Kemarin kendalanya itu data yang ada dalam GADM itu terlalu banyak sampai kecamatan. 
Saya hanya perlu data provinsinya saja secara keseluruhan. Rpubs yang dirujuk pun tipe yang berfokus pada 1 provinsi saja. DKI. Dan tidak merujuk ke beberapa provinsi sekaligus. 
Jadi sulit bagi saya untuk mengerti apalagi pengalaman coding saya pemula. 

Apakah ada cara lain untuk buat choroplet dengan dtabase csv?
Karna saya punya ternyata (baru buat sendiri).

**Reply by ranydwi** [reply]
Dear Bapak @Saupia 

Oke Bapak sebetulnya apabila Bapak merasa data GADM terlalu banyak sampai kecamatan, Bapak bisa mendowload untuk per-country. Sehingga data yang Bapak dapatkan nanti per-provinsi melalui [link berikut](https://gadm.org/download_country.html).

Kemudian, apabila Bapak telah memiliki data dalam format CSV (non spatial data) silahkan bapak convert data CSV ke SHP untuk medapatkan data spasialnya. Bapak dapat melakukan convertingnya secara online di google.

Sekian, semoga bermanfaat untuk Bapak.

Regards,
Team Algoritma
